.github/workflows/operator-ui-main.yml

```diff
@@ -0,0 +1,40 @@
+name: Operator UI
+
+on:
+  push:
+    branches:
+      - 'master'
```

Comment:
could we just have one xx-main file with all the tests?

Reply:
ah yes we can merge them

Reply:
hmm though the main reason was to group exactly the same tests - otherwise it's maybe less obvious that the ui tests are in both something-main and operator-ui

Reply:
yeah fair - I'm fine either way